syntax = "proto2";
package revolve.msgs;
import "time.proto";
import "pose.proto";

message RobotState {
  required uint32 id = 1;
  required string name = 2;
  required gazebo.msgs.Pose pose = 3;
}

message RobotStates {
  required gazebo.msgs.Time time = 1;
  repeated RobotState robot_state = 2;
}
